upload/management/commands/vcf_clean_and_filter.py

```diff
@@ -94,22 +98,24 @@ def handle(self, *args, **options):
                         continue
 
                 # Check ref / alt bases are ok
-                if settings.VARIANT_STANDARD_BASES_ONLY:
-                    ref = columns[3]
-                    alt = columns[4]
-                    if ref_standard_bases_pattern.sub("", ref):
-                        if ref.startswith("<") and ref.endswith(">"):
-                            skip_reason = f"REF = {ref}"
-                        else:
-                            skip_reason = "non-standard bases in REF sequence"
-                        skipped_records[skip_reason] += 1
-                        continue
+                ref = columns[3]
```

Comment:
I know it wasn't introduced in this PR, but when we reach 7 columns, would be good to have the columns defined by an (int, Enum) class rather than magic numbers.

Reply:
I tried doing it as an enum and either had to use .value or make a cast to int. If you just declare it as a boring old class you can do it easier IMO